diracx-core/src/diracx/core/settings.py

```diff
@@ -16,7 +16,7 @@
 if TYPE_CHECKING:
     from pydantic.config import BaseConfig
     from pydantic.fields import ModelField
-
+from cryptography.fernet import Fernet
```

Comment:
Purely for my curiosity as I haven't come across fernet before, is there a particular reason for having picked it?

Reply:
No particular reason, I just googled "python encryption" and found many examples of `Fernet`. It sounds very easy to use and well maintained.
I also noticed we were already using `cryptography` within the code.

Would you recommend any another library?

Reply:
We should definitely use something from `cryptography`. I would have defaulted to `AES-GCM-256` but it looks like [their docs recommend `Fernet`](https://cryptography.io/en/latest/) so that's good enough for me!